<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalSalaryClass</class>
 <widget class="QWidget" name="CalSalaryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalSalary</string>
  </property>
  <widget class="QLabel" name="preTaxLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>税前收入：</string>
   </property>
  </widget>
  <widget class="QLabel" name="fundBaseLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>公积金缴费基数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="socialBaseLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>社保缴费基数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="socialBaseRateLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>社保缴费比例：</string>
   </property>
  </widget>
  <widget class="QLabel" name="fundBaseRateLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>公积金缴费比例：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="preTaxIn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fundBaseIn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fundBaseRateIn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="socialBaseRateIn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="socialBaseIn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CalSalary.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>calButton</sender>
   <signal>clicked()</signal>
   <receiver>CalSalaryClass</receiver>
   <slot>StartCalSalary()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>-20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>StartCalSalary()</slot>
 </slots>
</ui>
